package.json

```diff
@@ -3,23 +3,29 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reach/router": "^1.3.1",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "@types/asana": "^0.14.37",
     "@types/jest": "^24.0.0",
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
     "antd": "^3.26.8",
+    "asana": "^0.17.3",
     "lodash": "^4.17.15",
     "moment": "^2.24.0",
     "react": "^16.12.0",
     "react-command-palette": "^0.10.0-0",
     "react-dom": "^16.12.0",
     "react-scripts": "3.3.1",
-    "typescript": "~3.7.2"
+    "react-use": "^13.24.0",
+    "typescript": "~3.7.2",
+    "use-persisted-state": "^0.3.0"
```

Comment:
This is pretty cool. I didn't know this existed, in most of our web app we use a similar pattern via Meteor's "Reactive Var" and "Session" so that we can achieve this. We could use this to replicate the `jiraState` pattern with react hooks without any provider gymnastics like with `useContext`. cc @bone-saw @BrunoBernardino

Reply:
It is a pretty convenient package. I just discovered it myself actually. I am finding that almost everything has a nice hook based approach now and I am loving it.

Reply:
I just browsed through their source code, there's no magic involved and I liked it! It's basically a cool "wrapper" around `localStorage` mapped to `useState`.